migrations/000048_HealthAuthorityStats.up.sql

```diff
@@ -0,0 +1,71 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+CREATE TABLE HealthAuthorityStatsRelease(
+    health_authority_id INT NOT NULL REFERENCES HealthAuthority(id),
+    not_before TIMESTAMPTZ NOT NULL,
+    -- timeone name, i.e. America/New_York
+    timezone TEXT,
+    -- last time that the timezone was changed - this locks the PHA out from changing again for configured time duration.
+    last_timezone_change TIMESTAMPTZ NOT NULL
+);
+
+CREATE TABLE HealthAuthorityStats(
+    health_authority_id INT NOT NULL REFERENCES HealthAuthority(id),
+    hour TIMESTAMPTZ NOT NULL,
+    publish INT NOT NULL DEFAULT 0,
+    teks INT NOT NULL DEFAULT 0,
+    revisions INT NOT NULL DEFAULT 0,
+    -- Age of the oldest TEKs from an individual publish request.
+    oldest_tek_0_days INT NOT NULL DEFAULT 0,
```

Comment:
I think it'd be better to use a [Postgres Array](https://www.postgresql.org/docs/9.1/arrays.html) here instead.

Reply:
changed to arrays, indexed by days